AlfrescoContent/AlfrescoContent/Classes/Swaggers/Models/GenericBucketBucketInfo.swift

```diff
@@ -15,13 +15,13 @@ public struct GenericBucketBucketInfo: Codable {
     /** The start of range */
     public var start: String?
     /** Includes values greater or equal to \&quot;start\&quot; */
-    public var startInclusive: Bool?
+    public var startInclusive: String?
```

Comment:
Have you pick this variable from Adf json?

Reply:
These are available in the response of search API.